### PR TITLE
Updated Python and Git links

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**

### CODE REVIEW

File: Python/experts_to_follow.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Name | Twitter | Google+ | Website/Blog | LinkedIn | Other links
 ------------ | ------------- | ------------- | ------------- | ------------- | ------------- |
-Guido van Rossum | https://twitter.com/gvanrossum | - | https://www.python.org/~guido/ | - | - 
+Guido van Rossum | https://twitter.com/gvanrossum | - | https://www.python.org/~guido/ | - | -
 Kenneth Reitz | https://twitter.com/kennethreitz | - | http://www.kennethreitz.org/ | - | https://github.com/kennethreitz
-David Beazley | https://twitter.com/dabeaz | - | http://www.dabeaz.com/ | - | - 
+David Beazley | https://twitter.com/dabeaz | - | http://www.dabeaz.com/ | - | -
 Armin Ronacher | https://twitter.com/mitsuhiko | - | http://lucumr.pocoo.org/ | - | https://github.com/mitsuhiko
-Adrian Holovaty | https://twitter.com/adrianholovaty | - | http://www.holovaty.com/ | - | https://github.com/adrianholovaty
```
Comment (PSanni) [inline Python/experts_to_follow.md]: What's new in this update ????? Unnecessary commit.

Comment (SubhrajyotiSen) [inline Python/experts_to_follow.md]: Added links for Miguel Grinberg (Python dev and writer for Flask)

Comment (PSanni) [inline Python/experts_to_follow.md]: :+1:

**Comment by PareshMayani** [comment]
@SubhrajyotiSen Hi Subhrajyoti, we have a conflict here, could you please resolve it?

**Comment by SubhrajyotiSen** [comment]
@PareshMayani resolved :)

**Comment by PareshMayani** [comment]
Thank you @SubhrajyotiSen for the quick resolution and contribution!

**Pull request merged**